src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -100,11 +100,27 @@ object EventMessages {
 
     override def slack(incomingEvent: ExportStatusEvent): Option[SlackMessage] = {
       if(incomingEvent.environment != "intg" || !incomingEvent.success) {
-        val message = s"The export for the consignment ${incomingEvent.consignmentId} has ${if (incomingEvent.success) "completed" else "failed"} for environment ${incomingEvent.environment}"
+
+        val exportInfoMessage = constructExportInfoMessage(incomingEvent)
+
+        val message = s"The export for the consignment ${incomingEvent.consignmentId} " +
+          s"has ${if (incomingEvent.success) "completed" else "failed"} for environment ${incomingEvent.environment}" +
+          s"${exportInfoMessage}"
         SlackMessage(List(SlackBlock("section", SlackText("mrkdwn", message)))).some
       } else {
         Option.empty
       }
     }
+
+    private def constructExportInfoMessage(incomingEvent: ExportStatusEvent): String = {
+     if (incomingEvent.successDetails.isDefined) {
```

Comment:
What happens if they're both defined, or neither? Does Circe allow you to use `Either` in this kind of situation?